.github/workflows/lint_dockerfile.yml

```
@@ -13,3 +13,23 @@ jobs:
         uses: hadolint/hadolint-action@54c9adbab1582c2ef04b2016b760714a4bfde3cf
         with:
           dockerfile: Dockerfile
+
+  dockle:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
+
+      - name: Build an image from the Dockerfile
+        run: |
+          docker build -t built-image:scan .
```

You could consider using the below code, which will use Docker layer caching to build the image, which should speed up things.

```suggestion
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@4c0219f9ac95b02789c1075625400b2acbff50b1
        with:
          install: true

      - name: Build Docker image and store in cache
        uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825
        with:
          context: .
          push: false
          load: true
          tags: exercism/zig-test-runner
          cache-from: type=gha
          cache-to: type=gha,mode=max
```

I don't know if the tag must be the same for the cache to work, but I figured just using the same tag could work just as well